acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/pubsub/WebSocketConnection.java

```diff
@@ -85,30 +85,33 @@ private JsonRpcSuccessEvent send(final String json) {
   private void connect(final Vertx vertx) {
     vertx
         .createHttpClient(new HttpClientOptions())
-        .websocket(
+        .webSocket(
```

Comment:
Lots of things named websocket, inconsistently, are now webSocket everywhere.

Reply:
++ consistency